### ISSUE TITLE
Custom protocol reported as invalid

### ISSUE THREAD

**Issue by hubertgajewski** [validator/validator#1949]
URL being validated or code to reproduce error:

```
<!DOCTYPE html>
<html lang="en">
<head>
<title>CSS URL demo</title>
</head>
<body>
<div style="background:url('foo://example.com/image.svg')"></div>
</body>
</html>
```

I did not find in HTML/CSS specification any rule which suggests that I can't use custom protocol.

**Comment by sideshowbarker** [comment]
That error is coming from the code for the CSS validator, which we just use as a library. So if you try with an equivalent CSS snippet at https://jigsaw.w3.org/css-validator/, I think you’ll find you can reproduce the error there. And if so, then you should raise an issue for it at https://github.com/w3c/css-validator/issues/new. And once any change gets made for it in the CSS validator code, then here we’ll just pick up that change.

**Comment by sideshowbarker** [comment]
Moved to https://github.com/w3c/css-validator/issues/466
